### PR TITLE
feat(frontend): ConvertInputsContainer component

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Component that will be re-used for Source and Destination tokens inputs.

Screenshot 2024-11-11 at 21 28 25
Screenshot 2024-11-11 at 21 28 44

### CODE REVIEW

File: src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="flex flex-col items-start justify-between md:flex-row">
+	<!-- TODO: Update colors with values from theme-variables when they are available there -->
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte:2]: just as info, there is this PR about it:

- #3444

File: src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	<div
+		class="mb-3 flex h-14 w-full items-center rounded-lg border-[1px] border-bright-gray bg-ghost-white px-3 md:mb-0 md:w-[35%]"
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte:4]: i think `border-[1px]` is just `border`

Comment (AntonioVentilii) [reply src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte:4]: plus, if 33.3% is ok too, you can use directly `w-4/12`

**Review by AntonioVentilii — APPROVED** [review]
LGTM, a few comments before merging

File: src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte:2]: 👍

File: src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte (hunk 2 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/convert/ConvertInputsContainer.svelte:4]: thanks!

**Pull request merged**